Changelog.md

```diff
@@ -16,7 +16,8 @@ Bugfixes:
  * Type Checker: Do not disallow assigning to calldata variables.
  * Wasm backend: Fix code generation for for-loops with pre statements.
  * Yul: Fix source location of variable multi-assignment.
-
+ * Yul Optimizer: LoopInvariantCodeMotion can move `extcodehash`, `extcodesize`, `returndatasize`, `balance` and  `selfbalance`, if the loop does not invalidate the blockchain state.
```

Comment:
Needs to move

Reply:
Maybe also summarize: move reading operations outside for loops as long as the affected area is not modified inside the loop